Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -2491,6 +2505,10 @@ namespace Babylon
                 std::set<xr::System::Session::Frame::InputSource::Identifier> added{};
```

Comment:
>std::set [](start = 16, length = 8)

Unless we deliberately want to keep these sorted for when we insert them into the NAPI arrays, these would probably be better served as a `std::deque` or `std::vector`.

Reply:
Yeah, I was just matching the usage below